lib/index-maplibre.js

```diff
@@ -4,9 +4,9 @@ var Plotly = require('./core');
 
 Plotly.register([
     // traces
-    require('./scattermapbox'),
-    require('./choroplethmapbox'),
-    require('./densitymapbox'),
+    require('./scattermaplibre'),
+    require('./choroplethmaplibre'),
+    require('./densitymaplibre'),
```

Comment:
I'm not sure we want to change traces names in data as well as mapbox subplot names in layout.
For now I suggest we keep `mapbox` in names.
That may also help introduce these updates as a minor change.

Reply:
Great idea, I'll move it back to that.